packages/app-builder-lib/package.json

```diff
@@ -59,6 +59,7 @@
     "builder-util": "workspace:*",
     "builder-util-runtime": "workspace:*",
     "chromium-pickle-js": "^0.2.0",
+    "ci-info": "^3.7.0",
```

Comment:
Can we update to a newer version of `ci-info`? [v4.2.0](https://github.com/watson/ci-info/releases/tag/v4.2.0)

In particular, [v3.9.0](https://github.com/watson/ci-info/releases/tag/v3.9.0) has Azure Signing for instance.

Reply:
done